docs/docs/guides/multitrack-progress.md

```diff
@@ -96,8 +96,9 @@ const setProgress = useProgressStateStore.getState().setProgress;
 export const PlaybackService = async function() {
   TrackPlayer.addEventListener(Event.PlaybackProgressUpdated, async () => {
     // get the position and currently playing track.
-    const position = TrackPlayer.getPosition();
-    const track = TrackPlayer.getCurrentTrack();
+    const position = await TrackPlayer.getPosition();
+    const trackIndex = await TrackPlayer.getCurrentTrack();
+    const track = await TrackPlayer.getTrack(trackIndex || 0);
```

Comment:
These two lines can now be simplified to:

```ts
const track = await TrackPlayer.getActiveTrack();
```

Reply:
done